nat-lab/tests/utils/network_switcher/network_switcher_windows.py

```diff
@@ -19,14 +20,23 @@ async def get_network_interfaces(connection: Connection) -> List["Interface"]:
             ["netsh", "interface", "ipv4", "show", "addresses"]
```

Comment:
We should remove `ipv4` here, as disabled interfaces does not show up then

Reply:
On the other hand, it will not show up only if it is disabled, which is ok in our case.